.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: c
+
+dist: trusty
+sudo: required
+
+compiler:
+  - gcc
+  - musl-gcc
+  - clang
+
+install:
+  # This ppa provides a more recent musl
+  - sudo add-apt-repository ppa:bortis/musl -y
+  - sudo apt-get update -qq
+  - sudo apt-get install -qq musl-tools musl musl-dev
+
+script:
+  - make 
```

Comment:
this can easily grow.  rather than do this, how about we create a `test/travis.sh` script to handle this logic and this line simply reads:

```
script: ./test/travis.sh
```

Reply:
I'd prefer a step-level in travis. Otherwise it can be difficult to figure out what failed on travis.

Reply:
that's purely an issue of the script and how it logs.  making it a standalone script makes it a lot easier to add extended/conditional logic as well as reproduce/execute the code locally.